Pack-Smart/pack-smart-fe
# Refactor/how it works

## What's this PR do?
- Completes responsiveness for `HowItWorks` component
- Separates reviews and steps into a data file `howItWorksData`
- Removes Pack Smart from `Header` and adds it to `Home` component

## Where should the reviewer start?
- `HowItWorks.js`, `howItWorksData.js`, `Header.js` and `Home.js`

## How should this be manually tested?
- View the `HowItWorks` component and view on multiple screen sizes to see responsiveness
- View the data file to see the separate data

## What are the relevant tickets?
- Closes #56 
- Closes #69 
- Closes #73

## Did you update the README?
- No, I did not
